src/components/Payroll/PayrollOverview/PayrollOverview.tsx

```diff
@@ -40,6 +42,22 @@ const findUnresolvedBlockersWithOptions = (blockers: PayrollSubmissionBlockersTy
   )
 }
 
+const findWireInRequestUuid = (
+  creditBlockers: PayrollCreditBlockersType[] = [],
+): string | undefined => {
+  const unresolvedCreditBlocker = creditBlockers.find(blocker => blocker.status === 'unresolved')
+
+  if (!unresolvedCreditBlocker?.unblockOptions) {
```

Comment:
is this because the API does not return `unblockOptions` at all if there are none, as opposed to an empty list?